web3rpc/rpc-specs/code-samples/javascript/admin/dataDirApi.js

```diff
@@ -1,8 +1,6 @@
-const OpenSdk = require("opensdk-javascript");
-
+const { JsonRpcProvider } = require("@klaytn/ethers-ext");
 (() => {
-    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));
-
+let sdk = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
```

Comment:
"provider" looks better instead of "sdk"

Reply:
Got it